client/src/main.rs

```diff
@@ -280,8 +280,6 @@ fn update_hosts_file(
     hosts_path: PathBuf,
     peers: &[Peer],
 ) -> Result<(), WrappedIoError> {
-    log::info!("updating {} with the latest peers.", "/etc/hosts".yellow());
```

Comment:
BTW I think the intention of @bschwind was to move (and reword) this info log, not to completely remove it?

Reply:
Right, we should print out a message if we actually wrote new data to the hosts file.

Reply:
Fixed!